e2e/rook-test-framework/clients/k8s_rook_object.go

```diff
@@ -0,0 +1,136 @@
+package clients
+
+import (
+	"errors"
+	"github.com/rook/rook/e2e/rook-test-framework/contracts"
+)
+
+type k8sRookObject struct {
+	transportClient contracts.ITransportClient
+}
+
+var (
+	objectStoreCrate         = []string{"rook", "object", "create"}
+	objectStoreGetBuckets    = []string{"rook", "object", "bucket", "list"}
+	objectStoreGetConnection = []string{"rook", "object", "connection", "REPLACE"}
+	objectStoreCreateUser    = []string{"rook", "object", "user", "create", "USERID", "DISPALYNAME"}
+	objectStoreDeleteUser    = []string{"rook", "object", "user", "delete", "USERID"}
+	objectStoreGetUser       = []string{"rook", "object", "user", "get", "USERID"}
+	objectStoreGetUsers      = []string{"rook", "object", "user", "list"}
+	objectStoreUpdateUser    = []string{"rook", "object", "user", "update", "USERID", "--display-name", "DISPLAYNAME",
+		"--email", "EMAILID"}
+)
+
+func CreateK8sRookObject(client contracts.ITransportClient) *k8sRookObject {
+	return &k8sRookObject{transportClient: client}
+}
+
+//Function to create a object store in rook
+//Input paramatres -None
+//Output - output returned by rook cli and/or error
+func (ro *k8sRookObject) Object_Create() (string, error) {
+	out, err, status := ro.transportClient.Execute(objectStoreCrate, nil)
+	if status == 0 {
+		return out, nil
+	} else {
+		return err, errors.New("Unabale to create Object store")
+	}
+}
+
+//Function to get Buckets present in rook object store
+//Input paramatres - None
+//Output - output returned by rook cli and/or error
+func (ro *k8sRookObject) Object_Bucket_list() (string, error) {
+	out, err, status := ro.transportClient.Execute(objectStoreGetBuckets, nil)
+	if status == 0 {
+		return out, nil
+	} else {
+		return err, errors.New("Unabale to get buckets")
+	}
+}
+
+//Function to get connection information for a user
+//Input paramatres - userId
+//Output - output returned by rook cli and/or error
+func (ro *k8sRookObject) Object_Connection(userid string) (string, error) {
+	objectStoreGetConnection[3] = userid
+	out, err, status := ro.transportClient.Execute(objectStoreGetConnection, nil)
+	if status == 0 {
+		return out, nil
+	} else {
+		return err, errors.New("Unabale to get buckets")
```

Comment:
this error message is repeated for most functions in this file.  they should be updated so they are accurate to the operation they are performing.